packages/transaction-controller/src/utils/transaction-type.ts

```diff
@@ -104,6 +113,12 @@ function parseStandardTokenTransactionData(
     // ignore and return undefined
   }
 
+  try {
+    return USDCInterface.parseTransaction({ data });
```

Comment:
Not needed now, but if we expect more of these in future, we could loop through a list of interfaces to remove some duplication.